mica-webapp/src/main/resources/_templates/libs/studies.ftl

```diff
@@ -55,33 +55,113 @@
-          <#else >
-            <p class="text-black-50 text-center mr-5 ml-5 pr-5">
-              <i class="${studyIcon} fa-3x"></i>
-            </p>
-          </#if>
+  </div><!-- /.card-header -->
+
+  <div class="card-body pb-0">
+    <div class="row">
+      <div class="col-12">
+        <div class="d-inline-flex float-right">
+          <sorting @sort-update="onSortUpdate" :initial-choice="initialSort" :options-translations="sortOptionsTranslations"></sorting>
+          <span class="ml-2 mr-1">
+                    <select class="custom-select" id="obiba-page-size-selector-top"></select>
+                  </span>
+          <nav id="obiba-pagination-top" aria-label="Top pagination" class="mt-0">
+            <ul class="pagination mb-0"></ul>
+          </nav>
         </div>
       </div>
     </div>
-    <div class="card-footer">
-      <div class="text-right">
-        <a href="${contextPath}/study/${study.id}" class="btn btn-sm btn-outline-info">
-          <i class="fas fa-eye"></i> <@message "global.read-more"/>
-        </a>
+  </div>
+
+  <div class="card-body pb-0">
+    <div class="tab-pane">
+      <div class="row d-flex align-items-stretch">
+        <div class="col-md-12 col-lg-6 d-flex align-items-stretch" v-for="study in entities" v-bind:key="study.id">
+          <div v-if="study.id === ''" class="card w-100">
+            <div class="card-body pt-0 bg-light">
+            </div>
+          </div>
+          <div v-else class="card w-100">
+            <div class="card-body">
+              <div class="row h-100">
+                <div class="col-xs-12 col">
+                  <h4 class="lead">
+                    <a v-bind:href="'${contextPath}/study/' + study.id" class="mt-2">
+                      <b>{{study.name | localize-string}}</b>
+                    </a>
+                  </h4>
+                  <span class="marked"><small :inner-html.prop="study.objectives | localize-string | ellipsis(300, ('${contextPath}/study/' + study.id)) | markdown"></small></span>
+                </div>
+                <div class="col-3 mx-auto my-auto" v-if="study.logo">
+                  <a v-bind:href="'${contextPath}/study/' + study.id" class="text-decoration-none text-center">
+                    <img class="img-fluid" style="max-height: 10em;" v-bind:alt="study.acronym | localize-string | concat(' logo')" v-bind:src="'${contextPath}/ws/study/' + study.id + '/file/' + study.logo.id + '/_download'"/>
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div class="card-footer py-1">
+              <div class="row pt-1 row-cols-4">
+                <template v-if="hasStats(study)">
+                  <a v-if="study.model && study.model.methods" href="javascript:void(0)" style="cursor: initial;" class="btn btn-sm col text-left">
```

Comment:
The items "Study Design" and "Number of Participants" are study model dependent and therefore should appear in a separate macro, so that they can be easily overridden.
(+ translation is missing)

Reply:
Is it wise to make it so granular? This is already inside of a macro: `studyCardModel`

Reply:
Yes it is, because it is model dependent and people are changing the MR's default model. It must be possible to modify model's related content without braking page functionalities.